src/pages/Home/Home.jsx

```diff
@@ -1,19 +1,26 @@
 import React from "react";
 import { FormattedMessage } from "react-intl";
-import { Grid } from "@mui/material";
+import { Grid, Typography, Box } from "@mui/material";
 import CountdownTimer from "../../components/CountdownTimer/CountdownTimer";
 import "../../components/CountdownTimer/CountdownTimer.css";
```

Comment:
:wrench: We should be importing a completely styled `CountdownTimer` component. No `CountdownTimer.css` is needed here.